axelor-account/src/main/resources/reports/AccountingReportType23.rptdesign

```diff
@@ -3000,7 +3000,7 @@ WHERE Invoice.status_select = 3 AND
                                     </list-property>
                                     <property name="contentType">html</property>
                                     <text-property name="content"><![CDATA[<VALUE-OF>
-if(params["Locale"].value == 'fr') {
+if(params["Locale"].value == 'fr_FR') {
```

Comment:
no you have to extract the language value in the Locale, it has to work even for fr_CA for example